VisualRecognitionV4.playground/Pages/Images.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,97 @@
+//:## Images
+
+import Foundation
+import VisualRecognitionV4
+
+let visualRecognition = setupVisualRecognitionV4()
+
+//:### Add images
+
+let trainingDataJSON = """
+{
+  "objects": [
+    {
+      "object": "2018-Fit",
+      "location": {
+        "left": 13,
+        "top": 5,
+        "width": 760,
+        "height": 419
+      }
+    }
+  ]
+}
+"""
```

Comment:
@mkistler FYI, all the other langs, including curl, just pass in a `training_data.json` file in the request example. We should switch Swift to that. (Say that 4 times fast!)

I also just added a parameter example (because we can with OAS 3!!) that shows the json string. Unfortunately, it shows up only for curl, but I'll create an issue for that. 

![image](https://user-images.githubusercontent.com/8194208/66678744-93984900-ec3a-11e9-94b5-169a64c73492.png)